WPMedia/fusion-news-theme-blocks
# fix: Add aria label to missing button per accessibility test

## Description
Add aria label for nav header button 

closes https://github.com/WPMedia/fusion-news-theme-blocks/issues/93

## Jira Ticket
- https://github.com/WPMedia/fusion-news-theme-blocks/issues/933 especially an issue with no visible text

## Acceptance Criteria
Shows aria-label for nav button 



## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout FIX-nav-header-aria-label`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/nav-header-chain-block`
3. Go to http://localhost/pf/all-blocks/?_website=arc-demo-5 and see non-english aria label for nav button 


## Effect Of Changes
### Before

![image](https://user-images.githubusercontent.com/5950956/128246347-83a038a6-19b9-4283-b5ec-80bf5c16995f.png)

### After
<img width="1122" alt="Screen Shot 2021-08-04 at 14 57 35" src="https://user-images.githubusercontent.com/5950956/128246471-fd29d72f-1261-4f7a-bdf0-5947380e9bd2.png">

## Dependencies or Side Effects


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
